cmd/clusterawsadm/api/bootstrap/v1alpha1/types.go

```diff
@@ -98,6 +98,8 @@ type ClusterAPIControllers struct {
 	// consumed by Cluster API when creating an ec2 instance. Defaults to
 	// *.<suffix>, where suffix is defaulted to .cluster-api-provider-aws.sigs.k8s.io
 	AllowedEC2InstanceProfiles []string `json:"allowedEC2InstanceProfiles,omitempty"`
+	// EnableEKS
+	EnableEKS bool `json:"enableEKS,omitempty"`
```

Comment:
Might be better to convert this into a struct in case additional options need to be added at a later date, so something like

```suggestion
	EKS EKSControllerConfig `json:"enableEKS,omitempty"`
```

with a struct somewhere
``` go
type EKSControllerConfig struct {
    Enable bool `json:"enable,omitempty"`
```